decentraland/screens/helpers.js

```diff
@@ -1,36 +1,175 @@
-import ContractsAddresses from "@constants/ContractsAddresses";
-const { MANA_ADDRESS } = ContractsAddresses;
+import TasitContracts from "tasit-contracts";
```

Comment:
Why not get it from `tasit-sdk`? We should either go fully modular or fully monolithic, I think

Reply:
`TasitContracts` isn't exposed by `TasitSDK` should be?
About that, I want to share a thought with you. For the contract that we are tracking the addresses, make sense we have a constructor interface without parameters that will get the address dynamically based on which network the user is?

Reply:
Yeah, I figured we'd expose all packages in the `tasit-sdk`.

Do you have any concerns about the size of the `tasit-contracts` package?

I really like that idea! Feel free to split that out into a separate `TasitSDK` issue or include it in an upcoming PR.

Reply:
1) Ok, the version v0.0.14 is already exposing it.
2) I think not, It's using 9.7% of the `tasit-sdk` size (https://bundlephobia.com/result?p=tasit-sdk@0.0.13)
3) Ok, thanks! Extracted to: https://github.com/tasitlabs/TasitSDK/issues/286

Reply:
1. Great.
2. Very cool! I had not seen that tool before. Oh yeah, I guess even in `0.0.13` it was still part of the bundle even if it wasn't exposed. I thought there might have been some fancy tree-shaking that would leave it out of the build files, but I guess not.
3. Perfect, thanks.

Reply:
Feel free to resolve this after reading

Reply:
Resolving this.